lib/tiny-jsonrpc/server.js

```diff
@@ -46,7 +46,7 @@ function functionSnippet(fn) {
 }
 
 function parseFunction(fn) {
-  var parsed = fn.toString().match(/function\s+(\w+)?\((.*)\)/);
+  var parsed = fn.toString().match(/function(?:\s+|\s+(\w+))?\s*\((.*)\)/);
```

Comment:
Can we just put the space inside the capture group instead? Like this: `/function(\s+\w+)?\((.*)\)/`. I guess we would need to trim in that case.